airbyte-tests/build.gradle

```diff
@@ -61,12 +61,6 @@ task acceptanceTests(type: Test) {
         exceptionFormat "full"
     }
     mustRunAfter test
-
-    dependsOn ':airbyte-integrations:connectors:source-postgres:airbyteDocker'
-    dependsOn ':airbyte-integrations:connectors:destination-postgres:airbyteDocker'
-    // for checkpointing tests
-    dependsOn ':airbyte-integrations:connectors:source-e2e-test:airbyteDocker'
```

Comment:
switching to pinned versions of these makes these unnecessary.

Reply:
nice!